libraries/lib-numeric-formats/ProjectTempoListenerInterface.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include <memory>
+
+class NUMERIC_FORMATS_API ProjectTempoListenerInterface
+{
+public:
+   virtual ~ProjectTempoListenerInterface();
+   virtual void SetProjectTempo(double newTempo) = 0;
```

Comment:
When you need an abstract interface with just one interesting virtual function -- just make an alias for a `std::function<void(double)>`